src/reduxTypes.js

```diff
@@ -256,6 +256,7 @@ export type RealmState = {|
   twentyFourHourTime: boolean,
   canCreateStreams: boolean,
   isAdmin: boolean,
+  isTopicMandatory: boolean,
```

Comment:
Let's keep this name very similar to the name in the API -- that helps simplify understanding what the code is doing, without the reader having to trace through all the layers.

The API's name for this is `realm_mandatory_topics`, and we're already on the "realm" state. So `mandatoryTopics` would be a good name.